src/remote/entrypoints.py

```diff
@@ -214,6 +214,14 @@ def remote_set(index: int):
     help="Enable local port forwarding. Pass value as <remote port>:<local port>. \
 If local port is not passed, the local port value would be set to <remote port> value by default",
 )
+@click.option(
+    "-s",
+    "--stream-changes",
+    default=False,
+    is_flag=True,
+    help="Continuously sync files from local to remote \
+     while executing the command",
```

Comment:
Let's make it a little more verbose and explain that it will only resync if changes were made